### DISCUSSION TITLE
"demo" Unit Ids

### DISCUSSION THREAD

**Discussion by kromtom** [Poing-Studios/godot-admob-editor#18] [Q&A] [answered]
Hi guys,

I have a question about the "demo" uint Ids that are automatically set up for different ad types. I tried to delete them and replace them with my units, but they keep coming back after every restart of the Godot editor.
Is this an intended behaviour? 

Thanks!

**Comment by gumaciel** [comment, marked as answer]
Hi @kromtom 

Depending on what you want to do is rather intended, by default the Editor will not allow you to delete the "demo", what you can do is change the value of "demo" to what you would like.

I think the form it is in might seem a little confusing (calling it demo), how about we change the name from "demo" to "standard"? So the "standard" key cannot be deleted and can be any value, including a demo Ad or a real Ad

**Reply by kromtom** [reply]
Thanks a lot, I get how it works now. Awesome module!
